lib/ts/recipe/accountlinking/recipe.ts

```diff
@@ -124,32 +112,80 @@ export default class Recipe extends RecipeModule {
         return SuperTokensError.isErrorFromSuperTokens(err) && err.fromRecipe === Recipe.RECIPE_ID;
     }
 
-    getIdentitiesForPrimaryUserId = async (
-        primaryUserId: string
+    getIdentitiesForUser = async (
```

Comment:
maybe rename this to `segregateVerifiedAndUnverifiedAccountInfo`? And return `{verified: AccountInfo[], unverified: AccountInfo[]}`?

Reply:
Disagree